src/main/java/com/uid2/admin/vertx/service/ClientKeyService.java

```diff
@@ -431,6 +437,39 @@ private void handleClientRoles(RoutingContext rc) {
         }
     }
 
+    private void handleClientRename(RoutingContext rc) {
```

Comment:
Can you add some tests to ClientKeyServiceTest? Should check if it works and should also check if it prevents a user from renaming to a non-unique name

Reply:
Done